### ISSUE TITLE
App is not published

### ISSUE THREAD

**Issue by sp1thas** [flathub/me.simakis.dropboxignore#4]
After the successful build of the CI/CD pipelines, [the app is still unpublished](https://flathub.org/apps/details/me.simakis.dropboxignore). Did I miss something in the config? Any clue @hfiguiere ? Thanks in advance.

**Comment by hfiguiere** [comment]
3 hours is the standard.

Also there are ongoing infra problem so it might take a longer time.

**Comment by sp1thas** [comment]
Still not published :'(

**Comment by hfiguiere** [comment]
@barthalion this failed to publish.

And right now can't rebuild because of https://github.com/flathub/flathub/issues/3870

**Comment by barthalion** [comment]
The app should be published shortly.

**Comment by sp1thas** [comment]
Reopen this one because the app is not published yet.

**Comment by barthalion** [comment]
It is published, since March last year.

```
~ 2.8s ❱ flatpak remote-info --user --log flathub me.simakis.dropboxignore
        ID: me.simakis.dropboxignore
       Ref: app/me.simakis.dropboxignore/x86_64/stable
      Arch: x86_64
    Branch: stable
Collection: org.flathub.Stable
  Download: 203,6 kB
 Installed: 231,9 kB
   Runtime: org.freedesktop.Platform/x86_64/21.08
       Sdk: org.freedesktop.Sdk/x86_64/21.08

    Commit: aa33ed8a18b15de86245d8e4583f88d3b4a4018f969bb27042e6e9ad1d815f04
    Parent: 112fbdab48eb5c659bf423e0761dd7fbb0e0bb3352acdc42a9dfa02b55442a97
   Subject: fix build-commands (ce29c410)
      Date: 2023-02-03 10:27:37 +0000
   History: 

    Commit: 112fbdab48eb5c659bf423e0761dd7fbb0e0bb3352acdc42a9dfa02b55442a97
   Subject: fix build-commands (ce29c410)
      Date: 2023-01-16 15:58:10 +0000

    Commit: c395e931a24f510818c752e4a54067683b68bf0b5cbe4400a4f7484b58e466c9
   Subject: fix build-commands (ce29c410)
      Date: 2023-01-15 22:00:04 +0000

    Commit: 453afc9bdfdefe23c6786a926caf8c214af67d6050ab4fe674e7b9b64ebc8205
   Subject: dummy change (352884ca)
      Date: 2022-03-15 19:26:58 +0000

    Commit: 7bd47fdb5ba1ce52a2601cb2221be88d13e20d773e3a6724619407ded653b6b0
   Subject: fix icons (25bd74ac)
      Date: 2022-03-15 07:06:33 +0000
```

We make it clear during review that CLI applications are not displayed on the website.

**Comment by sp1thas** [comment]
> We make it clear during review that CLI applications are not displayed on the website.

Didn't notice that @barthalion Thanks for letting me know.
